libs/database/src/lib/entities/delayed-job.entity.ts

```diff
@@ -78,37 +73,26 @@ export class DelayedJob extends Model<DelayedJob> implements EntityAssociations
 
   @BelongsTo(() => Project, {
     foreignKey: "entityId",
-    constraints: false,
-    scope: {
-      entityType: "App\\Models\\V2\\Projects\\Project"
-    },
-    as: "entityProject"
+    constraints: false
   })
   entityProject?: Project;
 
   @BelongsTo(() => Site, {
     foreignKey: "entityId",
-    constraints: false,
-    scope: {
-      entityType: "App\\Models\\V2\\Sites\\Site"
-    },
-    as: "entitySite"
+    constraints: false
```

Comment:
Why was the `entityType` scope removed? That, I believe, is required for this to function correctly.